jwst/skymatch/skymatch_step.py

```diff
@@ -216,3 +224,30 @@ def _set_sky_background(self, sky_image, library, step_status):
 
         dm.meta.cal_step.skymatch = step_status
         library.shelve(dm, index)
+
+
+    def _user_sky(self, library):
+        """Handle user-provided sky values for each image.
+        """
+
+        log.info(" ")
+        log.info("----  Using user-provided sky values for each image.")
+
+        if self.skylist is None:
+            raise ValueError('skymethod set to "user", but no sky values provided.')
+        if len(self.skylist) != len(library):
+            raise ValueError(f"Number of entries in skylist ({len(self.skylist)}) does not match "
+                             f"number of input images ({len(library)}).")
+
+        with library:
+            for i, model in enumerate(library):
```

Comment:
```suggestion
            self.log.info(f"Setting sky background of input images to user-provided values from `skylist` file '{self.skyfile}':")

            for i, model in enumerate(library):
```

Reply:
this was changed, resolving the conversation